Preferences.tsx

```diff
@@ -67,16 +69,20 @@ export const PreferencesProvider: React.FC<PreferencesProviderProps> = ({childre
     void AsyncStorage.setItem(PREFERENCES_KEY, JSON.stringify(preferences));
   }, [preferences]);
 
-  const setPreferences = (newPreferences: Partial<Preferences>) => {
-    setFullPreferences(merge({}, preferences, newPreferences));
+  const setPartialPreferences = (newPreferences: Partial<Preferences>) => {
```

Comment:
this is the function that gets publicly exposed, so that callers can easily set a single preference. no behavior change, it's just renamed internally to reflect that it accepts `Partial<Preferences>`.